opencti-platform/opencti-front/src/private/components/data/feeds/FeedCreation.jsx

```diff
@@ -168,6 +168,7 @@ const sharedUpdater = (store, userId, paginationOptions, newEdge) => {
 
 const FeedCreation = (props) => {
   const { t, classes } = props;
+  const { t_i18n } = useFormatter();
```

Comment:
if you are using `t_i18n` from `useFormatter` instead of `props.t`, you need to complete this refactor.

`compose(inject18n` (which was injecting the `t` prop) is no longer useful when using the hook `useFormatter`. And all mention to the `t` prop shall be removed.

Reply:
Done.. Thanks :)